pom.xml

```diff
@@ -16,6 +16,7 @@
 
 	<properties>
 		<java.version>11</java.version>
+		<version.spring-webmvc>5.2.6.RELEASE</version.spring-webmvc>
```

Comment:
> general remark - I think we should extract versions of central frameworks into properties in the pom so that you have a better overview on them and so that you can change them in one placer if they have more than one artifact

You suggested this earlier in your comment (see the above) and I agreed that it's a good idea to have all the versions in one place, hence the modification. 

No, I don't think `version.springframework` make sense here.